openlibrary/templates/type/author/view.html

```diff
@@ -220,9 +229,12 @@ <h3>$:_('Links <span class="gray small sansserif">(outside Open Library)')</span
 
         $if page.alternate_names:
             $if page.alternate_names != [""]:
-                <div class="section">
+                <div class="section" itemscope itemtype=$entityTypeSchema>
```

Comment:
This needs to get at the very top of the page, where you have `itemtype="https://schema.org/Person"`

```suggestion
                <div class="section">
```